app/services/csv_exporter.rb

```diff
@@ -0,0 +1,21 @@
+require 'csv'
+
+module CsvExporter
+  def self.to_csv(resources, columns)
+    Enumerator.new do |csv|
+      csv << csv_header(columns)
+
+      resources.find_each do |obj|
+        csv << csv_row(obj, columns)
```

Comment:
That example actually isn't good. Feel free to disregard until I provide better feedback.